x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/confirm_alert_save.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { EuiOverlayMask, EuiConfirmModal } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  onConfirm: () => void;
+  onCancel: () => void;
+}
+
+export const ConfirmAlertSave: React.FC<Props> = ({ onConfirm, onCancel }) => {
+  return (
+    <EuiOverlayMask>
+      <EuiConfirmModal
+        title={i18n.translate(
+          'xpack.triggersActionsUI.sections.confirmAlertSave.confirmAlertSaveTitle',
+          {
+            defaultMessage: 'Save Alert with no actions?',
+          }
+        )}
+        onCancel={onCancel}
+        onConfirm={onConfirm}
+        confirmButtonText={i18n.translate(
+          'xpack.triggersActionsUI.sections.confirmAlertSave.confirmAlertSaveConfirmButtonText',
+          {
+            defaultMessage: 'Save',
+          }
+        )}
+        cancelButtonText={i18n.translate(
+          'xpack.triggersActionsUI.sections.confirmAlertSave.confirmAlertSaveCancelButtonText',
+          {
+            defaultMessage: 'Cancel',
+          }
+        )}
+        defaultFocusedButton="confirm"
+        data-test-subj="confirmAlertSaveModal"
+      >
+        <p>
+          <FormattedMessage
+            id="xpack.triggersActionsUI.sections.confirmAlertSave.confirmAlertSaveMessage"
+            defaultMessage="Are you sure you want to save this alert without defining any actions?"
```

Comment:
@gchaps could you please verify the texts for a new confirmation dialog.

Reply:
The text "Are you sure" isn't necessary.  Can we come up with some text that describes the consequences of having an alert with no actions?

Reply:
@gchaps As I understand it, creating an alert with no actions means a user will need to return to the Alerting UI to determine if an alert has fired, they will not get notified in any other way. Does that sound correct @YulNaumenko @gmmorris @pmuellr? What wording would you suggest for this?

Reply:
yes, that's correct.  The general thought is that an alert without actions is kind of useless, but we don't want to prevent users from doing this, as there may be some valid use cases for it.

Somehow working in "if you don't have any actions, when the alerts fire, you will not be notified in any way" ... kinda thing.

Reply:
@gchaps How about "You will not be notified when when this alert fires".

Reply:
That's technically too specific - not all actions cause notifications, eg the index action.  However, trying to be more general is likely going to lead to a more confusing message, and certainly the first time experience for customers will be to use an actual notification, and I'd expect most actions customers use will be for notification purposes, so Works For Me.

Reply:
Maybe something simple like 'There is no actions attached to the alert. Do you want to save it?' 
I'm not sure that it is a part of the confirmation logic to let them know that they won't get any notifications will be a result of it, because a bunch of actions is not a notifications - like Index, ServiceNow, Jira, Resilient

Reply:
In the case that the user forgot to add an action, how about nudging them toward the "Create action" flyout.   

**Save alert with no actions?**

You can [add an action ](url)at anytime.

Cancel | Save alert